### PR TITLE
stm32mp15-eval lmp.cfg

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-eval-sec/lmp.cfg [hunk 1]
```diff
@@ -0,0 +1,26 @@
+CONFIG_BOOTCOMMAND="if test ${boot_device} = serial || test ${boot_device} = usb; then stm32prog ${boot_device} ${boot_instance}; else; setenv verify 1; source ${scriptaddr}; reset; fi;"
```
Comment (mike-scott) [inline meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-eval-sec/lmp.cfg:1]: On the -sec target boot command, we don't check for serial boot as there are mfgtools to handle provisioning.

**Review by mike-scott — CHANGES_REQUESTED** [review]

File: meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-eval-sec/lmp.cfg (hunk 1 above)
Comment (igoropaniuk) [reply meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-eval-sec/lmp.cfg:1]: right, will fix

**Review by MrCry0 — APPROVED** [review]

**Review by mike-scott — APPROVED** [review]
LGTM

**Pull request merged**